### ISSUE TITLE
AdditionalExecutable is not abstract and does not override abstract method executeOn

### ISSUE THREAD

**Issue by adriens** [sualeh/SchemaCrawler#151]
Hi @sualeh , since 14.17.01, plugin compilation is fialing with the following message :

Could you give us some guidelines on : 

- how to fix our code (plugin development best practive)
- confirm us that the plugin archetype has also been modified to take this new change in account 

Thank you in advance for your help, i've provided you the build log in case it could help :

```
[INFO] 2 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.511 s
[INFO] Finished at: 2017-11-20T13:25:59+11:00
[INFO] Final Memory: 20M/281M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.7.0:compile (default-compile) on project schemacrawler-dashboard: Compilation failure: Compilation failure:
[ERROR] /home/salad74/NetBeansProjects/schemacrawler-plugin-dashboard/src/main/java/com/github/adriens/AdditionalExecutable.java:[28,8] com.github.adriens.AdditionalExecutable is not abstract and does not override abstract method executeOn(schemacrawler.schema.Catalog,java.sql.Connection,schemacrawler.schemacrawler.DatabaseSpecificOverrideOptions) in schemacrawler.tools.executable.StagedExecutable
[ERROR] /home/salad74/NetBeansProjects/schemacrawler-plugin-dashboard/src/main/java/com/github/adriens/AdditionalExecutable.java:[48,17] executeOn(schemacrawler.schema.Catalog,java.sql.Connection) in com.github.adriens.AdditionalExecutable cannot override executeOn(schemacrawler.schema.Catalog,java.sql.Connection) in schemacrawler.tools.executable.BaseStagedExecutable
[ERROR] overridden method is final
```

**Comment by adriens** [comment]
```
public class AdditionalExecutable
        extends BaseStagedExecutable {

    private static final Logger LOGGER = Logger
            .getLogger(AdditionalExecutable.class.getName());
...



    @Override
    public void executeOn(final Catalog catalog, final Connection connection)
            throws Exception {
```

**Comment by sualeh** [comment]
@adriens I will leave this issue open until I can fix the archetype.

For your existing executable, you will need to override
```
  void executeOn(Catalog catalog,
                 Connection connection,
                 DatabaseSpecificOverrideOptions databaseSpecificOverrideOptions)
    throws Exception;
```

**Comment by adriens** [comment]
Thanks @sualeh , i give it a try right now !

**Comment by adriens** [comment]
Yep, it did the job. :+1:

**Comment by adriens** [comment]
Did you modify the archetyp with this new signature ?

**Comment by sualeh** [comment]
No, I am leaving this issue open until I fix the archetype.

**Comment by adriens** [comment]
Hmm, btw, i gues this is the objects that makes it possbile to user quotingStrategy, right ?

**Comment by adriens** [comment]
Impressive support performances : "adriens opened this Issue 2 hours ago " :smile_cat: 

Have a nice week !...and thanks for the release ! :rocket:

**Comment by sualeh** [comment]
@adriens - I am solving this another way, by not changing the original API for executables. Please wait for the next release.

**Comment by adriens** [comment]
Ok, i'm waiting for this.

**Comment by sualeh** [comment]
Sorry for breaking this 👎

**Comment by adriens** [comment]
No problem, i'm happy to be source of enhancement as bêta tester ;-p This a pretty cool collaboration process :+1:

**Comment by sualeh** [comment]
Fixed in SchemaCrawler 14.17.02.
